draft-packages/collapsible/KaizenDraft/ExpertAdviceCollapsible/ExpertAdviceCollapsible.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from "@testing-library/react"
+import { configure, queryByTestId } from "@testing-library/dom"
+import * as React from "react"
+import { ExpertAdviceCollapsible } from ".."
+
+configure({ testIdAttribute: "data-automation-id" })
+
+it("renders the title as a heading", () => {
+  const { getByTestId } = render(
+    <ExpertAdviceCollapsible id="1" title="This title should be rendered">
+      Expert advice collapsible component
+    </ExpertAdviceCollapsible>
+  )
+  const titleText = getByTestId("collapsible-header-1").querySelector("div")
+  expect(titleText).toHaveTextContent("This title should be rendered")
```

Comment:
In this instance we could alternatively use `screen.getByText()` within the expect to remove the dependence on a test id

Reply:
Updated! Thanks